ci-operator/jobs/openshift/release/openshift-release-release-4.2-periodics.yaml

```diff
@@ -848,3 +848,81 @@ periodics:
     - name: pull-secret
       secret:
         secretName: ci-pull-credentials
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: "12h"
+  labels:
+    job-env: aws
+    job-test: e2e
+    job-release: '4.2'
+  name: release-openshift-origin-installer-e2e-aws-4.2-cnv
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --give-pr-author-access-to-namespace=true
+      - --secret-dir=/usr/local/pull-secret
+      - --secret-dir=/usr/local/e2e-aws-cluster-profile
+      - --target=e2e-aws
+      - --template=/usr/local/e2e-aws
+      - --input-hash=$(BUILD_ID)
+      - --input-hash=$(JOB_NAME)
+      command:
+      - ci-operator
+      env:
+      - name: RELEASE_IMAGE_LATEST
+      - name: BRANCH
+        value: "4.2"
+      - name: CLUSTER_TYPE
+        value: aws
+      - name: CONFIG_SPEC
+        value: |
+          tag_specification:
+            name: "$(BRANCH)"
+            namespace: ocp
+          resources:
+            '*':
+              limits:
+                memory: 4Gi
+              requests:
+                cpu: 100m
+                memory: 200Mi
+          tests:
+          - as: e2e-$(CLUSTER_TYPE)
+            commands: TEST_SUITE=openshift/conformance/parallel run-tests
+            openshift_installer:
```

Comment:
Is this needed to perform the install?

Reply:
yes, this basically says "run this job using the cluster installer profile" which will spin up a new cluster for you before running the tests.

Reply:
this should be updated to use the openshift_custom_test_image api now, so you can point it to the new image you're defining.

Reply:
actually this content doesn't matter at all.  everything that is relevant comes from the template you're choosing to use (cluster-launch-installer-custom-test-image.yaml in this case).

Reply:
So can/should I remove the tests section completely?

Reply:
leave it for now, just be aware that it's not consumed by CI.